omnigibson/simulator.py

```diff
@@ -253,17 +280,24 @@ def __init__(
                 device=device,
             )
 
-            if self._world_initialized:
-                return
-            Simulator._world_initialized = True
-
             # Store other references to variables that will be initialized later
-            self._scene = None
-            self._physx_interface = None
-            self._physx_simulation_interface = None
-            self._physx_scene_query_interface = None
-            self._contact_callback = None
-            self._simulation_event_callback = None
+            self._scenes = []
+            self._physx_interface = lazy.omni.physx.get_physx_interface()
+            self._physx_simulation_interface = lazy.omni.physx.get_physx_simulation_interface()
+            self._physx_scene_query_interface = lazy.omni.physx.get_physx_scene_query_interface()
```

Comment:
so just to be clear, this never gets refreshed?

Reply:
Good question. @hang-yin can you check if this gets invalidated after play/stop and if it needs to be updated in update_handles? same for the interfaces above.

Reply:
Verify with clear